File: stress/t/ctl_condacc.c
#include <stdio.h>
#include <stdint.h>
int main(void){
  int64_t i=0,sum3=0,sum5=0,sumboth=0,cntev=0,x=0;
  for(i=1;i<=100;i=i+1){
    x=i;
    if(x%3==0){ sum3=sum3+x; } else { sum3=sum3+0; }
    if(x%5==0){ sum5=sum5+x; } else { sum5=sum5+0; }
    if(x%3==0 && x%5==0){ sumboth=sumboth+x; } else { sumboth=sumboth+0; }
    if(x%2==0){ cntev=cntev+1; } else { cntev=cntev+0; }
  }
  printf("sum3="); printf("%ld",(long)sum3); printf("\n");
  printf("sum5="); printf("%ld",(long)sum5); printf("\n");
  printf("sumboth="); printf("%ld",(long)sumboth); printf("\n");
  printf("cntev="); printf("%ld",(long)cntev); printf("\n");
  return 0;
}

File: stress/t/ctl_condacc.f
      PROGRAM CTLACC
      INTEGER*8 I,S3,S5,SB,CE,X
      S3=0
      S5=0
      SB=0
      CE=0
      I=1
   10 IF (I.GT.100) GOTO 20
      X=I
      IF (MOD(X,3_8).EQ.0) S3=S3+X
      IF (MOD(X,5_8).EQ.0) S5=S5+X
      IF ((MOD(X,3_8).EQ.0).AND.(MOD(X,5_8).EQ.0)) SB=SB+X
      IF (MOD(X,2_8).EQ.0) CE=CE+1
      I=I+1
      GOTO 10
   20 CONTINUE
      WRITE(*,'(A,I0)') 'sum3=',S3
      WRITE(*,'(A,I0)') 'sum5=',S5
      WRITE(*,'(A,I0)') 'sumboth=',SB
      WRITE(*,'(A,I0)') 'cntev=',CE
      END
